### PR TITLE
Fix beta build

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -2,3 +2,7 @@ github:
   branch_name: main
   tooling_branch_name: main
 conda_forge_output_validation: true
+
+build_platform:
+  osx_arm64: osx_64
+test: native_and_emulated
```
Comment (peastman) [inline conda-forge.yml:8]: Are you sure that's really what you want?  Is it going to be able to run tests when cross compiling?

Comment (mikemhenry) [reply conda-forge.yml:8]: I was following this blog post: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/

I've never tried to manually get the osx_arm builds to fire off manually, I always use the bot method.

What makes this annoying is the only reason we can't keep this package `noarch` (which would make osx_arm support work out of the box) is the selector we need to make NNPops not a dependency for osx...

Maybe we just go back to `noarch` and remove the nnpops dependency entirely? We can add a note in the README that if you are on linux, you should install nnpops. 

Thoughts @peastman ?

Comment (peastman) [reply conda-forge.yml:8]: > I've never tried to manually get the osx_arm builds to fire off manually, I always use the bot method.

I was just referring to the `test` option.  Should it be set to `native`?

> Maybe we just go back to noarch and remove the nnpops dependency entirely?

In the short term we can probably work with that.  In the long term, we'll need some way to make sure it gets installed on Linux, or people will end up getting much slower performance.

**Comment by mikemhenry** [comment]
@peastman I think this is what we want, `noarch` so we don't need to worry about having different builds, and I removed `nnpops` so this package will install on osx and linux just fine. 

In the hosted env for linux, I'll make sure to include `nnpops` so people will be able to get that package.

**Comment by peastman** [comment]
Even though you added the `rc` labels to `channel_sources`, the log shows it's still building against 7.7.  I think that's probably related to the problem in #3?  The version number for the beta of this package should be 1.0beta rather than 0.0.1.

**Comment by peastman** [comment]
Any insight on those issues?

**Comment by mikemhenry** [comment]
I'll try pinning the openmm version, I think the solver is having a problem

**Comment by mikemhenry** [comment]
Since this is noarch and linux, I can build this one locally to reduce troubleshooting time...

**Comment by mikemhenry** [comment]
@peastman there was an issue with the version string (needed to be `openmm==8.0.0beta`) New issue is interesting:
`package openmm-8.0.0beta-py37hd99be19_0 requires python >=3.7,<3.8.0a0`

**Comment by mikemhenry** [comment]
Now I am getting `package openmm-torch-0.1-py38hfe53b8f_3 requires openmm >=7.5.0,<7.6.0a0, but none of the providers can be installed`

**Comment by peastman** [comment]
It ought to be specifying version 1.0beta for openmm-torch.  0.1 is a very old build from over a year ago.

**Comment by mikemhenry** [comment]
I'm now testing (locally) specifying 1.0beta

**Comment by mikemhenry** [comment]
Once we get builds all sorted, I will update the hosted envs to specify the versions of everything so 1) we can be sure the right versions are pulled and 2) when users report errors, it should tell us exactly what is missing

**Comment by mikemhenry** [comment]
Maybe once the new openmm 8 beta builds hit the cdn, things will improve...

**Comment by peastman** [comment]
Any progress on this?

I'm also trying to figure out how to test locally.  But having just spent an hour trying without success to get Docker to work on my computer, I'm not optimistic about that happening any time soon.

**Comment by peastman** [comment]
I finally managed to get docker working on a different computer.  Now I can run the build locally and reproduce the problem.  But I'm kind of at a loss for how to debug it.  The error happens inside `conda mambabuild` which seems to be an opaque operation.

@mikemhenry I could really really use your help on this.

**Comment by mikemhenry** [comment]
@peastman I asked about this issue on the conda-forge gitter 
image (1)

So I can reproduce it locally, but not sure why it fails.

@jaimergp do you have any ideas?

**Comment by mikemhenry** [comment]
@peastman this should be good to merge now, the tl;dr is this PR makes the package noarch, so then we don't even need to worry about having arm builds.

**Comment by peastman** [comment]
Excellent!  Let's try building, then update the environments and see if it works.

**Comment by peastman** [comment]
Apparently I don't currently have permission to merge it, so you'll need to.

**Comment by mikemhenry** [comment]
I've added you to the list! Now I just need to update the hosted env and see if it works on linux and mac!

**Pull request merged**